invenio_app_ils/eitems/mappings/v6/eitems/eitem-v1.0.0.json

```diff
@@ -33,69 +33,7 @@
         "document": {
           "properties": {
             "authors": {
-              "properties": {
-                "affiliations": {
-                  "properties": {
-                    "identifiers": {
-                      "properties": {
-                        "scheme": {
-                          "type": "keyword"
-                        },
-                        "value": {
-                          "type": "keyword"
-                        }
-                      },
-                      "type": "object"
-                    },
-                    "name": {
-                      "type": "text",
-                      "fields": {
-                        "keyword": {
-                          "type": "keyword"
-                        }
-                      }
-                    }
-                  },
-                  "type": "object"
-                },
-                "alternative_names": {
-                  "type": "text",
-                  "fields": {
-                    "keyword": {
-                      "type": "keyword"
-                    }
-                  }
-                },
-                "full_name": {
-                  "type": "text",
-                  "fields": {
-                    "keyword": {
-                      "type": "keyword"
-                    }
-                  }
-                },
-                "identifiers": {
-                  "properties": {
-                    "material": {
-                      "type": "keyword"
-                    },
-                    "scheme": {
-                      "type": "keyword"
-                    },
-                    "value": {
-                      "type": "keyword"
-                    }
-                  },
-                  "type": "object"
-                },
-                "roles": {
-                  "type": "keyword"
-                },
-                "type": {
-                  "type": "keyword"
-                }
-              },
-              "type": "object"
+              "type": "text"
```

Comment:
I think it was a good decision not resolve the authors with the whole structure :)